Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -224,6 +211,23 @@ extension AlertManager {
 // MARK: Extensions
 
 extension FileManager {
+    
+    fileprivate func ensureDirectoryExists(_ directory: URL?, log: DiagnosticLog) {
+        guard let directory = directory else {
```

Comment:
[nit] Make `directory` non-optional? Seems odd to be calling a method named `ensureDirectoryExists` and pass it nil for the directory.

Reply:
It's for unit testing